frontend/src/libraries/explorer-wamp/accounts.ts

```diff
@@ -88,7 +88,18 @@ export default class AccountsApi extends ExplorerApi {
       };
     };
 
-    const queryTransactionCount = async () => {
+    const [accountInfo, accountBasic] = await Promise.all([
+      this.queryAccount(accountId),
+      queryBasicInfo(),
+    ]);
+    return {
+      ...accountInfo,
+      ...accountBasic,
+    };
+  }
+
+  async queryTransactionCount(accountId: string): Promise<AccountStats> {
```

Comment:
I suggest we rename the function to reflect the relation to the account (e.g. `queryAccountStats`). This way the name of the function and the return value make sense.

P.S. update the `state` field name in the AccountDetails component accordingly.